src/components/views/rooms/RoomPreviewBar.tsx

```
@@ -586,7 +586,23 @@ export default class RoomPreviewBar extends React.Component<IProps, IState> {
             if (!Array.isArray(subTitle)) {
                 subTitle = [subTitle];
             }
-            subTitleElements = subTitle.map((t, i) => <p key={`subTitle${i}`}>{t}</p>);
+
+            if( messageCase === MessageCase.Kicked ){
+                subTitleElements = subTitle.map((t: string, i: number) => (
+                    <p
+                        key={`subTitle${i}`}
+                        dangerouslySetInnerHTML={{
+                            __html: t.replace(
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)